### PR TITLE
build: fix api guardian issues on windows

### PR DESCRIPTION
**Pull request by alan-agius4**
This is a workaround for https://github.com/bazelbuild/rules_nodejs/issues/317

### CODE REVIEW

**Comment by alan-agius4** [comment]
//cc @alexeagle

File: tools/ts-api-guardian/lib/serializer.ts [hunk 1]
```diff
@@ -145,6 +145,10 @@ class ResolvedDeclarationEmitter {
     return output;
   }
 
+  private isExportPatternStripped(symbolName: string): boolean {
+    return [].concat(this.options.stripExportPattern).some(p => !!(p && symbolName.match(p)));
```
Comment (alexeagle) [inline tools/ts-api-guardian/lib/serializer.ts]: use `[...this.options.stripExportPattern]` to clone the array?

File: tools/ts-api-guardian/lib/cli.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export function startCli() {
   const {argv, mode, errors} = parseArguments(process.argv.slice(2));
 
   const options: SerializationOptions = {
-    stripExportPattern: argv['stripExportPattern'],
+    stripExportPattern: [].concat(argv['stripExportPattern']),
```
Comment (alexeagle) [inline tools/ts-api-guardian/lib/cli.ts]: use array spread to clone the array?

**Review by alexeagle — APPROVED** [review]

File: tools/ts-api-guardian/lib/cli.ts (hunk 2 above)
Comment (alan-agius4) [reply tools/ts-api-guardian/lib/cli.ts]: This is not to clone it, but to cat it to an array if it’s not

File: tools/ts-api-guardian/lib/serializer.ts (hunk 1 above)
Comment (alan-agius4) [reply tools/ts-api-guardian/lib/serializer.ts]: Same as above.

Comment (alan-agius4) [reply tools/ts-api-guardian/lib/serializer.ts]: Same as above.

**Comment by alan-agius4** [comment]
Care taker: if it doesn’t apply cleanly on the patch branch please target only master.
Thanks

**Pull request closed without merging**